lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/schema/JsonSchema.json

```diff
@@ -83,11 +83,11 @@
 			"type": "array",
 			"description": "See: <a href=\"https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.1.2\">enum</a>",
 			"items": {
-				"type": "string"
+				"type": "object"
```

Comment:
Maybe let us try to setup a silly test with an enum of values different from string, e.g. {"test": { "type": "enum", values: [1, 2, 3]}}